docs/admin/maintenance/upgrade/upgrade-4.0.0.md

```diff
@@ -0,0 +1,195 @@
+---
+sidebar_position: 2
+id: upgrade-4.0.0
+title: Upgrade 4.0.0
+description: Upgrading to 4.0.0
+draft: false
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Upgrading to OpenCloud 4.0.0
+
+This guide describes how to upgrade OpenCloud to the stable version
+**v4.0.0** using the Git repository [opencloud-compose](https://github.com/opencloud-eu/opencloud-compose.git)
+with externalized data and config directories.
+
+## Stop OpenCloud
+
+First, gracefully stop your currently running OpenCloud instance:
+
+<Tabs>
+
+<TabItem value="docker" label="docker">
+
+```bash
+docker stop opencloud
```

Comment:
Are you sure? It feels redundant to use docker and docker compose commands.

Is there a specific reason why to stop a single container first before stopping the compose stack?
If yes, that needs to be explained.

Reply:
no, we use here tabs `docker` and `docker compose`

it looks like
<img width="622" height="297" alt="image" src="https://github.com/user-attachments/assets/96db1897-1b98-464d-8b90-bbfa2911292a" />